packages/plans-grid/src/plans-table/index.tsx

```diff
@@ -29,32 +31,85 @@ const PlansTable: React.FunctionComponent< Props > = ( {
 	onPickDomainClick,
 	currentDomain,
 } ) => {
+	const { __ } = useI18n();
+
 	const supportedPlans = useSelect( ( select ) => select( PLANS_STORE ).getSupportedPlans() );
 	const prices = useSelect( ( select ) => select( PLANS_STORE ).getPrices() );
-	const [ allPlansExpanded, setAllPlansExpanded ] = useState( false );
+
+	// Primary plan
+	// TODO: Feature picker determines recommended plan depending on the features that user choose.
+	const recommendedPlanSlug = null;
+	const popularPlanSlug = 'value_bundle';
```

Comment:
We should get this value from the plans data-store.
`useSelect( ( select ) => select( PLANS_STORE ).getDefaultPaidPlan() );`

Reply:
Going to roll this together with another PR coming up where `useRecommendedPlan()` is used.